package.json

```diff
@@ -31,20 +44,57 @@
         "webpack-dev-server": "^4.7.3"
     },
     "dependencies": {
-        "@types/jest": "^27.4.0",
-        "@types/node": "^17.0.14",
+        "@istanbuljs/nyc-config-typescript": "^1.0.2",
         "@types/react": "^17.0.38",
         "@types/react-dom": "^17.0.11",
         "axios": "^0.25.0",
+        "babel-plugin-istanbul": "^6.1.1",
```

Comment:
I think `babel-plugin-istanbul` and `@istanbuljs/nyc-config-typescript` are used in development
Need to check if all works fine with these libs in devDependencies

Reply:
yes it is dev and not dep